main.go

```diff
@@ -43,6 +48,32 @@ func setEnv() {
 		logrus.SetLevel(logrus.DebugLevel)
 	}
 
+	var err error
+
+	// initialize polling and forceUpdate intervals
+	i = os.Getenv(EnvVarForceUpdateInterval)
+	if i == "" {
+		logrus.Info(EnvVarForceUpdateInterval + " is not set, using default value '1'")
+		i = "1"
```

Comment:
I would keep the default values up in the const section, not defining the default values in code here.

Reply:
agree... I was following poor precedent set by targetPoolSuffix ... I'll pull all 3 up into const as defaults.